src/amo/reducers/experiments.js

```diff
@@ -0,0 +1,51 @@
+/* @flow */
+export type StoredVariant = {| id: string, variant: string |};
+
+export const STORE_EXPERIMENT_VARIANT: 'STORE_EXPERIMENT_VARIANT' =
+  'STORE_EXPERIMENT_VARIANT';
+
+export type ExperimentsState = {
+  storedVariant: StoredVariant | null,
+};
```

Comment:
I would have expected a map of experiments where the key is the experiment ID and the variant is the associated value. When the map is empty, there is no experiment.

Reply:
This was a first-pass, proof-of-concept, so, as discussed in a comment I made, it's only set up to work with one experiment at a time. I do think that's something we should change, and that will result in a state similar to the one you suggest. I think that may address some of your other comments as well, so I will make that change.